Windows/ppsspp.rc

```diff
@@ -444,7 +451,7 @@ BEGIN
         MENUITEM "Control Mapping...",                      ID_OPTIONS_CONTROLS
         MENUITEM "Language...",                             ID_OPTIONS_LANGUAGE
         MENUITEM SEPARATOR
-        MENUITEM "Stretch to Display",                      ID_OPTIONS_STRETCHDISPLAY
+		MENUITEM "Display Layout Editor"                    ID_OPTIONS_DISPLAY_LAYOUT

```

Comment:
Doesn't really matter, but this file is spaces-land for some msvc-editor reason.

-[Unknown]